### PR TITLE
Counterbalance the biped air-time reward on Cassie/Go2/H1

### PR DESCRIPTION
**Pull request by Double7sBurger**
# Description

Cassie/H1/Go2 velocity tasks converge on a badly lopsided gait: one leg stays planted while the other
hangs, which reads as "standing on one leg" in playback. 

`feet_air_time_positive_biped` scores the length of the current single-stance phase and takes the
minimum over the two feet:

```python
in_mode_time  = torch.where(in_contact, contact_time, air_time)
single_stance = torch.sum(in_contact.int(), dim=1) == 1
reward = torch.min(torch.where(single_stance.unsqueeze(-1), in_mode_time, 0.0), dim=1)[0]
reward = torch.clamp(reward, max=threshold)
```

Nothing in it distinguishes left from right or asks the feet to alternate. Holding one foot planted
and the other airborne lets both timers run and sits at the clamp, while a symmetric gait scores lower.

This PR adds `feet_air_time_variance`, the variance across feet of the last completed swing and
stance durations, and enables it on all the three tasks. It vanishes when both feet are loaded
equally, so it prices the exploit without constraining anything else.

## Type of change

- New feature (non-breaking change which adds functionality)

Policies trained on these two tasks will differ from ones trained before this change. Setting
`env.rewards.air_time_variance.weight=0.0` recovers the previous reward exactly.

## Release backport

- [ ]  Backport this pull request to the active release branch after it merges into `develop`

## Checklist

- [x] I have read and understood the [contribution guidelines](https://isaac-sim.github.io/IsaacLab/main/source/refs/contributing.html)
- [x] I have run the [`pre-commit` checks](https://pre-commit.com/) with `./isaaclab.sh --format`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added a changelog fragment under `source/<pkg>/changelog.d/` for every touched package
- [x] I have added my name to the `CONTRIBUTORS.md` or my name already exists there

### CODE REVIEW

**Review by AntoineRichard — COMMENTED** [review]
LGTM!

**Review by AntoineRichard — APPROVED** [review]

**Comment by AntoineRichard** [comment]
@Double7sBurger can you make separate PRs when you fix different bugs in the future. Let's avoid making bundle PRs with a lot of different unrelated fixes

**Comment by Double7sBurger** [comment]
> @Double7sBurger can you make separate PRs when you fix different bugs in the future. Let's avoid making bundle PRs with a lot of different unrelated fixes

OK got it.
